### ISSUE TITLE
getting long training time  200000 steps and its only 1 model on 1 girl ! 

### ISSUE THREAD

**Issue by MosaabRkia** [sangyun884/HR-VITON#79]
what wrong with this how i change steps of that 

(Try) C:\Users\abork\Desktop\yes>python train_generator.py --cuda True --name Train1 -b 1 -j 1 --gpu_ids 0 --tocg_checkpoint gen.ckpt --occlusion
Namespace(name='Train1', gpu_ids=[0], workers=1, batch_size=1, fp16=False, cuda='True', dataroot='./data/', datamode='train', data_list='train_pairs.txt', fine_width=768, fine_height=1024, radius=20, grid_size=5, tensorboard_dir='tensorboard', checkpoint_dir='checkpoints', tocg_checkpoint='gen.ckpt', gen_checkpoint='', dis_checkpoint='', tensorboard_count=100, display_count=100, save_count=10000, load_step=0, keep_step=100000, decay_step=100000, shuffle=False, lpips_count=1000, test_datasetting='paired', test_dataroot='./data/', test_data_list='test_pairs.txt', G_lr=0.0001, D_lr=0.0004, GMM_const=None, semantic_nc=13, gen_semantic_nc=7, norm_G='spectralaliasinstance', norm_D='spectralinstance', ngf=64, ndf=64, num_upsampling_layers='most', init_type='xavier', init_variance=0.02, no_ganFeat_loss=False, no_vgg_loss=False, lambda_l1=1.0, lambda_feat=10.0, lambda_vgg=10.0, n_layers_D=3, netD_subarch='n_layer', num_D=2, GT=False, occlusion=True, warp_feature='T1', out_layer='relu', clothmask_composition='warp_grad', num_test_visualize=3)
Start to train Train1!
Network [SPADEGenerator] was created. Total number of parameters: 100.5 million. To see the architecture, do print(network).
Network [MultiscaleDiscriminator] was created. Total number of parameters: 1.3 million. To see the architecture, do print(network).
Setting up Perceptual loss...
C:\Users\abork\Desktop\yes\Try\lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
C:\Users\abork\Desktop\yes\Try\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=AlexNet_Weights.IMAGENET1K_V1`. You can also use `weights=AlexNet_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Loading model from: C:\Users\abork\Desktop\yes\eval_models\weights\v0.1\alex.pth
...[net-lin [alex]] initialized
...Done
C:\Users\abork\Desktop\yes\Try\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG19_Weights.IMAGENET1K_V1`. You can also use `weights=VGG19_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
  0%|                                                                                       | 0/200000 [00:00<?, ?it/s]C:\Users\abork\Desktop\yes\Try\lib\site-packages\torch\nn\functional.py:4296: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  warnings.warn(
C:\Users\abork\Desktop\yes\train_generator.py:302: UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\torch\csrc\tensor\python_tensor.cpp:85.)
  GAN_Feat_loss = torch.cuda.FloatTensor(len(opt.gpu_ids)).zero_()
  0%|                                                                          | 4/200000 [01:19<1124:23:46, 20.24s/it]

**Comment by veerabrahmam24** [comment]
Hey hi MosaabRKia,

Can you please let me know that how much GPU you have allocated to train this...? @MosaabRKia
